plugins/tasks.py

```diff
@@ -0,0 +1,138 @@
+#
+# coding=utf-8
+# flake8: noqa E302
+"""Development related tasks to be run with 'invoke'.
+
+Make sure you satisfy the following Python module requirements if you are trying to publish a release to PyPI:
+    - twine >= 1.11.0
+    - wheel >= 0.31.0
+    - setuptools >= 39.1.0
+"""
+import os
+import re
+import shutil
+import sys
+
+import invoke
+from plugins.ext_test import tasks as ext_test_tasks
+
+# create namespaces
+namespace = invoke.Collection()
+namespace_clean = invoke.Collection('clean')
+namespace.add_collection(namespace_clean, 'clean')
+
+#####
+#
+# pytest, tox, pylint, and codecov
+#
+#####
+
+
+@invoke.task()
+def pytest(_, junit=False, pty=True):
```

Comment:
This task is broken for me, I get a traceback like the following one:
```sh
inv pytest 
Traceback (most recent call last):
  File "/Users/toddleonhardt/src/cmd2/.venv/bin/inv", line 8, in <module>
    sys.exit(program.run())
  File "/Users/toddleonhardt/src/cmd2/.venv/lib/python3.8/site-packages/invoke/program.py", line 384, in run
    self.execute()
  File "/Users/toddleonhardt/src/cmd2/.venv/lib/python3.8/site-packages/invoke/program.py", line 566, in execute
    executor.execute(*self.tasks)
  File "/Users/toddleonhardt/src/cmd2/.venv/lib/python3.8/site-packages/invoke/executor.py", line 129, in execute
    result = call.task(*args, **call.kwargs)
  File "/Users/toddleonhardt/src/cmd2/.venv/lib/python3.8/site-packages/invoke/tasks.py", line 127, in __call__
    result = self.body(*args, **kwargs)
  File "/Users/toddleonhardt/src/cmd2/plugins/tasks.py", line 36, in pytest
    iexec.execute([
  File "/Users/toddleonhardt/src/cmd2/.venv/lib/python3.8/site-packages/invoke/executor.py", line 88, in execute
    calls = self.normalize(tasks)
  File "/Users/toddleonhardt/src/cmd2/.venv/lib/python3.8/site-packages/invoke/executor.py", line 154, in normalize
    name, kwargs = task
ValueError: not enough values to unpack (expected 2, got 1)
```

Reply:
I need to just delete this. I was trying to see if I could trick invoke into passing the command-line parameters down.